.github/workflows/wheels-dependencies.sh

```diff
@@ -1,31 +1,86 @@
 #!/bin/bash
 
-# Setup that needs to be done before multibuild utils are invoked
-PROJECTDIR=$(pwd)
-if [[ "$(uname -s)" == "Darwin" ]]; then
-    # Safety check - macOS builds require that CIBW_ARCHS is set, and that it
-    # only contains a single value (even though cibuildwheel allows multiple
-    # values in CIBW_ARCHS).
+# Safety check - Pillow builds require that CIBW_ARCHS is set, and that it only
+# contains a single value (even though cibuildwheel allows multiple values in
+# CIBW_ARCHS). This check doesn't work on Linux because of how the CIBW_ARCHS
+# variable is exposed.
+function check_cibw_archs {
     if [[ -z "$CIBW_ARCHS" ]]; then
-        echo "ERROR: Pillow macOS builds require CIBW_ARCHS be defined."
+        echo "ERROR: Pillow builds require CIBW_ARCHS be defined."
         exit 1
     fi
     if [[ "$CIBW_ARCHS" == *" "* ]]; then
-        echo "ERROR: Pillow macOS builds only support a single architecture in CIBW_ARCHS."
+        echo "ERROR: Pillow builds only support a single architecture in CIBW_ARCHS."
         exit 1
     fi
+}
+
+# Setup that needs to be done before multibuild utils are invoked. Process
+# potential cross-build platforms before native platforms to ensure that we pick
+# up the cross environment.
+PROJECTDIR=$(pwd)
+if [[ "$CIBW_PLATFORM" == "ios" ]]; then
+    check_cibw_archs
+    # On iOS, CIBW_ARCHS is actually a multi-arch - arm64_iphoneos,
+    # arm64_iphonesimulator or x86_64_iphonesimulator. Split into the CPU
+    # platform, and the iOS SDK.
+    PLAT=$(echo $CIBW_ARCHS | sed "s/\(.*\)_\(.*\)/\1/")
+    IOS_SDK=$(echo $CIBW_ARCHS | sed "s/\(.*\)_\(.*\)/\2/")
+
+    # Build iOS builds in `build/iphoneos` or `build/iphonesimulator/`
```

Comment:
```suggestion
    # Build iOS builds in `build/iphoneos` or `build/iphonesimulator`
```
Consistency. Either both should end in a slash, or neither.